### DISCUSSION TITLE
Easiest way to install OvenMediaEngine : OME Docker Launcher

### DISCUSSION THREAD

**Discussion by getroot** [OvenMediaLabs/OvenMediaEngine#1205] [Announcement]
To make it easier for users to install and operate OvenMediaEngine, we distribute OME Docker Launcher.

We know that many users have had trouble modifying Server.xml or loading certificates when using Docker. With OME Docker Launcher, you can do it all with a single command line.

```
$ curl -OL 'https://raw.githubusercontent.com/AirenSoft/OvenMediaEngine/master/misc/ome_docker_launcher.sh' && chmod +x ome_docker_launcher.sh
$ ./ome_docker_launcher.sh -h

 ▄██████▀███▄
█████▀ ▄██████  OvenMediaEngine Launcher v0.1
███▄▄▄▄▀▀▀▀███
██████▀ ▄█████  https://github.com/AirenSoft/OvenMediaEngine
 ▀███▄██████▀

• Usage: ./ome_docker_launcher.sh [OPTIONS] COMMAND ...

• Options:
  -h, --help                        Show this help message and exit
  -v, --version                     Show the version and exit
  -d, --debug                       Show debug log
  -b, --hide_banner                 Hide the banner
  -m, --monochrome                  Disable colors

• Commands:
  setup       Download the latest Docker image and setup directories for the container
  start       Start a docker container
  sh          Run a shell in the docker container
  status      Show the status of the docker container
  stop        Stop the docker container
  restart     Restart the docker container
```

![image](https://user-images.githubusercontent.com/17779085/234848244-ef4aff1a-a58c-4acd-aae1-eb90c000993e.png)

Please refer to the manual for details.

https://airensoft.gitbook.io/ovenmediaengine/v/master-1/getting-started/getting-started-with-ome-docker-launcher

**Comment by naanlizard** [comment]
Only just now found this, love the ASCII banner :)

**Comment by mauricioguimaraes-br** [comment]
Cannot start OME via launcher.  Every time I get this error:
• Starting OvenMediaEngine...
• Obtaining the port list from /usr/share/ovenmediaengine/conf/Server.xml
awk: line 55: syntax error at or near ,
• There is no Bind list. Please check the settings.

**Reply by dimiden** [reply]
The current script is written based on `gawk`, so the issue you mentioned occurs when using `awk`. Therefore, until this issue is resolved, I recommend installing and running `gawk` if possible.

**Reply by mauricioguimaraes-br** [reply]
Solved installing gawk. Thanks a lot!
